test/e2e_env/kubernetes/healthcheck/virtual_probes.go

```diff
@@ -0,0 +1,62 @@
+package virtual_probes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kumahq/kuma/test/e2e_env/kubernetes/env"
+	. "github.com/kumahq/kuma/test/framework"
+	"github.com/kumahq/kuma/test/framework/deployments/testserver"
+)
+
+func VirtualProbes() {
+	const name = "test-server"
+	const namespace = "virtual-probes"
+	const mesh = "virtual-probes"
+
+	BeforeAll(func() {
+		E2EDeferCleanup(func() {
+			Expect(env.Cluster.TriggerDeleteNamespace(namespace)).To(Succeed())
+			Expect(env.Cluster.DeleteMesh(mesh))
+		})
+
+		err := NewClusterSetup().
+			Install(MeshKubernetes(mesh)).
```

Comment:
Doesn't VirtualProbes only make sense with mTLS?

Reply:
right, sounds like a mistake of original test